systemd-user/icyci-instance.conf

```diff
@@ -0,0 +1,23 @@
+# Git URL for the repository under test (required)
+SOURCE_REPO="git://example.com/my/source-repo"
+
+# Git branch for the repository under test
+SOURCE_BRANCH="source_branch"
+
+# Test script path, relative to source-repo or absolute (required)
+TEST_SCRIPT="./test.sh"
+
+# Git URL to push test results to (required)
+RESULTS_REPO="ssh://icyci@example.com/my/results-repo"
+
+# Push source-branch and any tag to results-repo, in addition to notes.
+# If the corresponding source branch is not pushed, then cloning the
+# results repository only (if different to source) may lead to
+# confusion due to the missing commits referenced by the notes.
+PUSH_SOURCE_TO_RESULTS="true"
+
+# While idle, poll source-repo for changes at this `seconds` interval
+POLL_INTERVAL="60"
+
+# use TMPDIR (Go's os.TempDir) as a temporary workspace
+#TMPDIR=/tmp
```

Comment:
One other thing I missed - this matches the existing `systemd/icyci-instance.conf` file. I'd prefer to drop the duplicate and just point the documentation at `systemd/icyci-instance.conf`.
In fact, we should probably just put the new service file in the same `systemd` directory (e.g. `systemd/icyci-user@.service`).